ngraph/test/onnx/onnx_import_deprecated.in.cpp

```diff
@@ -0,0 +1,49 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <algorithm>
+#include <cmath>
+#include <cstdint>
+#include <fstream>
+#include <iterator>
+#include <limits>
+#include <numeric>
+#include <sstream>
+#include <stdexcept>
+#include <vector>
+
+#include "gtest/gtest.h"
+#include "ngraph/ngraph.hpp"
+#include "onnx_import/onnx.hpp"
+#include "util/all_close.hpp"
+#include "util/all_close_f.hpp"
+#include "util/engine/test_engines.hpp"
+#include "util/ndarray.hpp"
+#include "util/test_case.hpp"
+#include "util/test_control.hpp"
+#include "util/test_tools.hpp"
+
+using namespace ngraph;
+
+static std::string s_manifest = "${MANIFEST}";
+
+using TestEngine = test::ENGINE_CLASS_NAME(${BACKEND_NAME});
+
+using Inputs = std::vector<std::vector<float>>;
+using Outputs = std::vector<std::vector<float>>;
+
+NGRAPH_TEST(${BACKEND_NAME}, onnx_model_affine)
```

Comment:
@ilyachur could you please remind me. Do we still extend this tests? or we always should use singe-layer tests?

Reply:
At the current moment, we are migrating backend tests to reference tests. I propose try to move these tests to template plugin too.